### DISCUSSION TITLE
I don't have college email id , but i have proof as a student of college , what should i do ?

### DISCUSSION THREAD

**Discussion by PHCoder05** [community/community#139888] [GitHub Education]
### Hi everyone,

![Screenshot_3](https://github.com/user-attachments/assets/e0a61258-bccc-42f3-b7f8-8d3167e3ba49)
why it is showing mandatory to provide me college email id ?

**Comment by davevad93** [comment]
Hi @PHCoder05 ,

To apply for GitHub Education benefits, it's mandatory to use an academic email, some years ago it was possible to [submit an application without having one](https://github.blog/developer-skills/career-growth/how-to-get-the-github-student-developer-pack-without-a-student-id/), but now I believe it's no longer possible. You could contact support but they no longer provide manual reviews, however If you believe your school's details are incorrect, you can open a ticket [here](https://support.github.com/contact/education), and select `My selected school has incorrect or incomplete information e.g. domains or campus location`.

**Comment by Taiyorath** [comment]
same here if u don't have any college provided email id it is very difficult to have education account in github
